### ISSUE TITLE
Support and test generator self-scheduling into the market

### ISSUE THREAD

**Issue by adowling2** [grid-parity-exchange/Prescient#16]
Is this current supported in Prescient?

**Comment by bknueven** [comment]
No, but it should be easy to add once we switch over to Egret -- i.e., a self-schedule should look like a no-cost fixed commitment with p_min == p_max.
